[2022-04-07 19:20:53,569] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: load_to_BigQuery.create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task scheduled__2014-06-01T00:00:00+00:00 [queued]>
[2022-04-07 19:20:53,600] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: load_to_BigQuery.create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task scheduled__2014-06-01T00:00:00+00:00 [queued]>
[2022-04-07 19:20:53,600] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-04-07 19:20:53,601] {taskinstance.py:1250} INFO - Starting attempt 1 of 1
[2022-04-07 19:20:53,601] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-04-07 19:20:54,414] {taskinstance.py:1270} INFO - Executing <Task(BigQueryInsertJobOperator): create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task> on 2014-06-01 00:00:00+00:00
[2022-04-07 19:20:54,514] {standard_task_runner.py:52} INFO - Started process 2166 to run task
[2022-04-07 19:20:54,598] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'load_to_BigQuery', 'create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task', 'scheduled__2014-06-01T00:00:00+00:00', '--job-id', '12633', '--raw', '--subdir', 'DAGS_FOLDER/load_to_BigQuery_dag.py', '--cfg-path', '/tmp/tmpcn9rk36d', '--error-file', '/tmp/tmpbug95hr2']
[2022-04-07 19:20:54,650] {standard_task_runner.py:80} INFO - Job 12633: Subtask create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task
[2022-04-07 19:20:55,159] {logging_mixin.py:109} INFO - Running <TaskInstance: load_to_BigQuery.create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task scheduled__2014-06-01T00:00:00+00:00 [running]> on host 2c54f2187010
[2022-04-07 19:20:55,557] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=load_to_BigQuery
AIRFLOW_CTX_TASK_ID=create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task
AIRFLOW_CTX_EXECUTION_DATE=2014-06-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2014-06-01T00:00:00+00:00
[2022-04-07 19:20:55,559] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-04-07 19:20:55,612] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-04-07 19:20:55,656] {bigquery.py:1554} INFO - Inserting job ***_load_to_BigQuery_create_external_posts_answers_bq_create_posts_answers_stack_overflow_data_partitioned_table_task_2014_06_01T00_00_00_00_00_ca0d488b89724fb758996812de29c459
[2022-04-07 19:20:56,495] {taskinstance.py:1774} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 2153, in execute
    job = self._submit_job(hook, job_id)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 2119, in _submit_job
    retry=self.result_retry,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/common/hooks/base_google.py", line 439, in inner_wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 1560, in insert_job
    job.result(timeout=timeout, retry=retry)
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/job/query.py", line 1498, in result
    do_get_result()
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/job/query.py", line 1488, in do_get_result
    super(QueryJob, self).result(retry=retry, timeout=timeout)
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/job/base.py", line 728, in result
    return super(_AsyncJob, self).result(timeout=timeout, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/google/api_core/future/polling.py", line 135, in result
    raise self._exception
google.api_core.exceptions.BadRequest: 400 Could not cast literal "creation_date" to type TIMESTAMP at [1:109]

Location: us-central1
Job ID: airflow_load_to_BigQuery_create_external_posts_answers_bq_create_posts_answers_stack_overflow_data_partitioned_table_task_2014_06_01T00_00_00_00_00_ca0d488b89724fb758996812de29c459

[2022-04-07 19:20:56,547] {taskinstance.py:1288} INFO - Marking task as FAILED. dag_id=load_to_BigQuery, task_id=create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task, execution_date=20140601T000000, start_date=20220407T192053, end_date=20220407T192056
[2022-04-07 19:20:56,594] {standard_task_runner.py:98} ERROR - Failed to execute job 12633 for task create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task (400 Could not cast literal "creation_date" to type TIMESTAMP at [1:109]

Location: us-central1
Job ID: airflow_load_to_BigQuery_create_external_posts_answers_bq_create_posts_answers_stack_overflow_data_partitioned_table_task_2014_06_01T00_00_00_00_00_ca0d488b89724fb758996812de29c459
; 2166)
[2022-04-07 19:20:56,644] {local_task_job.py:154} INFO - Task exited with return code 1
[2022-04-07 19:20:58,412] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-04-07 22:14:26,894] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: load_to_BigQuery.create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task scheduled__2014-06-01T00:00:00+00:00 [queued]>
[2022-04-07 22:14:26,968] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: load_to_BigQuery.create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task scheduled__2014-06-01T00:00:00+00:00 [queued]>
[2022-04-07 22:14:26,968] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-04-07 22:14:26,968] {taskinstance.py:1250} INFO - Starting attempt 1 of 1
[2022-04-07 22:14:26,968] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-04-07 22:14:27,574] {taskinstance.py:1270} INFO - Executing <Task(BigQueryInsertJobOperator): create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task> on 2014-06-01 00:00:00+00:00
[2022-04-07 22:14:27,598] {standard_task_runner.py:52} INFO - Started process 4407 to run task
[2022-04-07 22:14:27,658] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'load_to_BigQuery', 'create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task', 'scheduled__2014-06-01T00:00:00+00:00', '--job-id', '13886', '--raw', '--subdir', 'DAGS_FOLDER/load_to_BigQuery_dag.py', '--cfg-path', '/tmp/tmp42i2v3ck', '--error-file', '/tmp/tmp_787dhk7']
[2022-04-07 22:14:27,660] {standard_task_runner.py:80} INFO - Job 13886: Subtask create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task
[2022-04-07 22:14:27,937] {logging_mixin.py:109} INFO - Running <TaskInstance: load_to_BigQuery.create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task scheduled__2014-06-01T00:00:00+00:00 [running]> on host 36107d30f64e
[2022-04-07 22:14:28,179] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=load_to_BigQuery
AIRFLOW_CTX_TASK_ID=create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task
AIRFLOW_CTX_EXECUTION_DATE=2014-06-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2014-06-01T00:00:00+00:00
[2022-04-07 22:14:28,181] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-04-07 22:14:28,218] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-04-07 22:14:28,246] {bigquery.py:1554} INFO - Inserting job ***_load_to_BigQuery_create_external_posts_answers_bq_create_posts_answers_stack_overflow_data_partitioned_table_task_2014_06_01T00_00_00_00_00_1bfbe7b2879f356c389ba43c198e4539
[2022-04-07 22:14:31,755] {taskinstance.py:1288} INFO - Marking task as SUCCESS. dag_id=load_to_BigQuery, task_id=create_external_posts_answers.bq_create_posts_answers_stack_overflow_data_partitioned_table_task, execution_date=20140601T000000, start_date=20220407T221426, end_date=20220407T221431
[2022-04-07 22:14:31,865] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-07 22:14:31,980] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
